### DISCUSSION TITLE
password protection for browser ?

### DISCUSSION THREAD

**Discussion by revolverocelot1** [zanllp/infinite-image-browsing#449] [Ideas]
i sometimes host stable diffusion public UI on site. users sometimes, misuse it .so it might be nice to have a password protection to access file? . because its hard to delete and add an extension on that platform as i don't have file access, every time someone fucks up.  i i know very specific and niche request but might be helpful in future.

**Comment by zanllp** [comment]
You can take a look at this link first, it might help you solve your problem.

https://github.com/zanllp/sd-webui-infinite-image-browsing/blob/main/.env.example

**Reply by revolverocelot1** [reply]
thanks, also is there any way i can put pictures or file , through this extension?

**Reply by zanllp** [reply]
There is no way unless you implement it yourself.

**Reply by revolverocelot1** [reply]
i wish i had that talent
